specificatie/openapi.yaml

```diff
@@ -648,6 +828,94 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoekobject om te zoeken op geometrie.'
+      type: object
+      properties:      
+        geometrie:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
+        spatialOperator:
+          type: string
+          description: 'Een uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1'
+          enum:
+#            - bbox
+#            - equals
+#            - disjoint
+            - intersects
+#            - touches
+#            - crosses
+            - within
+            - contains
+#            - overlaps
+#            - beyond
+#            - dwithin
+        tolerantie:
```

Comment:
heb je hier een description bij? Wat betekent tolerantie hier? Wat is de consequntie voor het zoekresultaat?

Reply:
OK